### ISSUE TITLE
ChromeOS-Dark-Oceanic-3.36 checkbox is not displayed

### ISSUE THREAD

**Issue by Durlyn** [rtlewis88/Solarized-Project#24]
Hi!
you can make the checkbox appear in windows?
![image](https://user-images.githubusercontent.com/874902/81262615-d2fb0c80-9046-11ea-91d7-97532a96d515.png)

**Comment by rtlewis88** [comment]
What operating system version are you installing the 3.36 on?

**Comment by Durlyn** [comment]
Fedora 31 and now Fedora 32

**Comment by rtlewis88** [comment]
For some reason, Fedora 32 wants the other variant (not the 3.36). All of our 3.36 problems have solely been with Fedora 32 lately - but fortunately they too have all been solved by switching to the other variant. That's what I would do.

**Comment by Durlyn** [comment]
in the theme solarized-dark-orange there is no such problem.

**Comment by rtlewis88** [comment]
Yep, that's a Numix-based theme. But, it looks like the material themes need to stick with the non-3.36 for now, on Fedora 32.
